java/runtime/src/main/java/io/ray/runtime/task/TaskExecutor.java

```diff
@@ -165,7 +166,7 @@ private RayFunction getRayFunction(List<String> rayFunctionInfo) {
         boolean isCrossLanguage = parseFunctionDescriptor(rayFunctionInfo).signature.equals("");
         if (hasReturn || isCrossLanguage) {
           returnObjects.add(ObjectSerializer
-              .serialize(new RayTaskException("Error executing task " + taskId, e)));
```

Comment:
If `RayTaskException` is not used anymore, can you delete it?

Reply:
OK. I will delete the `RayTaskException`.